lib/Dialect/Torch/Transforms/LowerToBackendContract.cpp

```diff
@@ -285,19 +285,16 @@ class LowerToBackendContractPass
   }
 };
 
-class VerifyBackendContractPass
-    : public VerifyBackendContractBase<VerifyBackendContractPass> {
+class VerifyBackendContractNoDecompositionsPass
+    : public VerifyBackendContractNoDecompositionsBase<VerifyBackendContractNoDecompositionsPass> {
 public:
-  VerifyBackendContractPass() = default;
-  VerifyBackendContractPass(bool decompose,
-                            ArrayRef<std::string> backendLegalOps) {
-    this->decompose = decompose;
-    this->backendLegalOps = backendLegalOps;
-  }
+  VerifyBackendContractNoDecompositionsPass() = default;
+
   void runOnOperation() override {
     MLIRContext *context = &getContext();
     ConversionTarget target =
-        getBackendContractTarget(context, decompose, backendLegalOps);
+        getBackendContractTarget(context, /*decompose*/false,
```

Comment:
nit: these comments should have an `=`, to match the style. In other words, it should be `/*decompose=*/`. Same for `backendLegalOps`

Reply:
Im working on another PR now to cleanup some changes. So let me fix this to in this PR.